docs/windows.md

```diff
@@ -0,0 +1,35 @@
+# Installing Calypso on Windows
+
+The set up for Windows is a bit more complicated than if you were on a \*nix system. This is because Node.js does not build anymore under Cygwin or MinGW and as such we are stuck with the Windows build which uses Windows-style paths (`C:\Windows\system`), while most of the utilities we use use Unix style paths (`/home/stuff/here`) and they don't play well together. For this reason it's very important to follow the installation steps carefully, so that correct versions of the needed utilities are installed.
+
+1. *When installing, it's best not to use folders with spaces in them.*
+2. Install [Git for Windows](https://git-scm.com/download/win). Make sure to check the "Use Git from the Windows Command Prompt" during installation!
+3. Install [Node.js](https://nodejs.org/en/download/) through the normal Windows installer.
+4. Install [MSYS2](https://msys2.github.io/).
+5. Make sure to follow the installation instructions for MSYS2 and update:
+
+    ```
+    pacman --needed -Sy bash pacman pacman-mirrors msys2-runtime
+    # restart MSYS2
+    pacman -Su
+    ```
+6. For the following commands and generally for working with Calypso, use the `MSYS2 Shell` that's been added to your Start menu.
+7. Install `make`:
+
+    ```
+    pacman -S make
+    ```
+8. Make sure that `git` is available and the Windows version is used (do *not* install `git` from MSYS2):
+
+    ```
+    which git
+	/c/Git/cmd/git
```

Comment:
When I install Git from the link above, and select the following values, my path looks like `/c/Program Files/Git/cmd/git`.

Prompt selections:
![screenshot 3](https://cloud.githubusercontent.com/assets/1126811/12861836/d2330714-cc2b-11e5-8d53-9775f29586ed.png)
![screenshot 4](https://cloud.githubusercontent.com/assets/1126811/12861833/d2311058-cc2b-11e5-9dcc-5acda47fda63.png)
![screenshot 5](https://cloud.githubusercontent.com/assets/1126811/12861834/d2328384-cc2b-11e5-8a2e-5f3e29a6a5f3.png)
![screenshot 6](https://cloud.githubusercontent.com/assets/1126811/12861837/d2339756-cc2b-11e5-87aa-5bee4c588909.png)
![screenshot 7](https://cloud.githubusercontent.com/assets/1126811/12861835/d232b1ec-cc2b-11e5-8183-366ef6efd768.png)
![screenshot 9](https://cloud.githubusercontent.com/assets/1126811/12861854/fcd2132a-cc2b-11e5-8443-66612f3ffac4.png)
![screenshot 10](https://cloud.githubusercontent.com/assets/1126811/12861874/21b31f72-cc2c-11e5-9938-9cdf7e857160.png)